app/assets/stylesheets/common.scss

```diff
@@ -504,7 +504,13 @@ body.small-nav {
 @include color-mode(dark) {
   .leaflet-tile-container,
   .mapkey-table-entry td:first-child > * {
-    filter: brightness(.8);
+    filter: invert(100%) hue-rotate(180deg) brightness(95%) contrast(90%);
```

Comment:
It doesn't look exactly the same, but whether it's better or worse is probably a matter of personal preference. I don't care.

Reply:
But is it really a big deal? How old would the hardware have to be for this to visibly impact performance? I have no issues with my even longer filter on a 10 years old desktop CPU. So unless you have a specific reason to worry about this, I would ignore that comment.

Reply:
Telling people to ignore comments based on your own anecdotal experience isn't exploring the issue. I have first hand experience of performance drop on older smart phones using multiple filters.

Reply:
@Dr-Mx My bad, I worded my comment poorly and I've edited it since then. I meant to say that you should give us more information about the problems you've encountered. Otherwise it would be hard to make an informed decision. Performance is important, so it's good that you mentioned this, but how old were those smartphones?

Reply:
I have tried using a similar inversion filter on my own website `invert(90%) hue-rotate(180deg) contrast(80%);` (looks best imho), Upon testing this with various devices, there was a noticeable stuttering when panning the map with low-end/older devices.
It would be good to do some proper testing, especially with devices in 3rd world countries where phones aren't as powerful as ours. Or simply have dark-mode as an option, not default.

Reply:
This PR isn't about proper testing, it's about fixing what has been rolled out without any.

That said, I agree with you that performance needs to be considered. However, that's gonna require more solid numbers than "I tested it on my device and it was stuttering".

Reply:
No PR should be accepted without testing.